src/scene_battle_rpg2k.cpp

```diff
@@ -371,6 +371,11 @@ void Scene_Battle_Rpg2k::ProcessActions() {
 }
 
 bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase* action) {
+	// Immediately quit for dead actors no move. Prevents any animations or delays.
+	if (action->GetType() == Game_BattleAlgorithm::Type::NoMove && action->GetSource()->IsDead()) {
+		return true;
```

Comment:
you access the action here before the nullptr check

Reply:
nullptr check not needed, it's gone.